test/hypothesis/DrPerceptronHypothesisTest.py

```diff
@@ -11,40 +11,40 @@ def tearDown(self):
         self._hypothesis = None
 
     def test_attacked_bad(self):
-        """Test the aggro == bad case"""
+        """Perceptron: Test the aggro == bad case"""
         for i in range(100):
             self._hypothesis.update([1],1,-1)
         trainedGuess = self._hypothesis.get_guess([1])
-        print ('DRPERCEPTRON: test_attacked_bad TrainedGuess: '+str(trainedGuess))
-        print (str(self._hypothesis))
+        #print ('DRPERCEPTRON: test_attacked_bad TrainedGuess: '+str(trainedGuess))
+        #print (str(self._hypothesis))
         self.assertTrue(trainedGuess==False)
 
     def test_attacked_good(self):
-        """Test the non aggro == good case"""
+        """Perceptron: Test the non aggro == good case"""
         for i in range(100):
             self._hypothesis.update([1],1,1)
         trainedGuess = self._hypothesis.get_guess([1])
-        print ('DRPERCEPTRON: test_attacked_good TrainedGuess: '+str(trainedGuess))
-        print (str(self._hypothesis))
+        #print ('DRPERCEPTRON: test_attacked_good TrainedGuess: '+str(trainedGuess))
+        #print (str(self._hypothesis))
```

Comment:
I was using this for debugging.  Since it's a test file, I thought it would still be useful to keep them in here; albeit commented.